app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    before_action :configure_permitted_parameters, if: :devise_controller?
+
+    protected
+
+    def configure_permitted_parameters
+      configure_signup_sanitizer
```

Comment:
It feels redundant to configure both of them every time. Try smth like `... if action_name == 'new'`

Reply:
@andrewr224 @lessuk400 I believe we can go with `request.put?`, `request.post?`,.. if those actions uses different request methods.